index.html

```diff
@@ -38,6 +39,156 @@
+            <li class="html-box"><p class="code-name">html</p></li>
+            <li class="bootstrap-box"><p class="code-name">bootstrap</p></li>
+            <li class="ruby-box"><p class="code-name">Ruby</p></li>
+          </ul>
+          
+       </div>
+       <div class="worktwo">
+        <p class="work-two-title">Profesional Art Printing Data</p>
+        <p class="work-two-dis">A daily selection of privately personalized reads; no accounts 
+          or sign-ups required. has been the industry's standard</p>
+          <div class="see-two-box"><p class="see-two">see Project</p></div>
+          <ul class="languages-work">
+            <li class="html-box"><p class="code-name">html</p></li>
+            <li class="bootstrap-box"><p class="code-name">bootstrap</p></li>
+            <li class="ruby-box"><p class="code-name">Ruby</p></li>
+          </ul>
+          
+       </div>
+       <div class="worktwo">
+        <p class="work-two-title">Profesional Art Printing Data</p>
+        <p class="work-two-dis">A daily selection of privately personalized reads; no accounts 
+          or sign-ups required. has been the industry's standard</p>
+          <div class="see-two-box"><p class="see-two">see Project</p></div>
+          <ul class="languages-work">
+            <li class="html-box"><p class="code-name">html</p></li>
+            <li class="bootstrap-box"><p class="code-name">bootstrap</p></li>
+            <li class="ruby-box"><p class="code-name">Ruby</p></li>
+          </ul>
+          
+       </div>
+       <div class="worktwo">
+        <p class="work-two-title">Profesional Art Printing Data</p>
+        <p class="work-two-dis">A daily selection of privately personalized reads; no accounts 
+          or sign-ups required. has been the industry's standard</p>
+          <div class="see-two-box"><p class="see-two">see Project</p></div>
+          <ul class="languages-work">
+            <li class="html-box"><p class="code-name">html</p></li>
+            <li class="bootstrap-box"><p class="code-name">bootstrap</p></li>
+            <li class="ruby-box"><p class="code-name">Ruby</p></li>
+          </ul>
+          
+       </div>
+       <div class="worktwo">
+        <p class="work-two-title">Profesional Art Printing Data</p>
+        <p class="work-two-dis">A daily selection of privately personalized reads; no accounts 
+          or sign-ups required. has been the industry's standard</p>
+          <div class="see-two-box"><p class="see-two">see Project</p></div>
+          <ul class="languages-work">
+            <li class="html-box"><p class="code-name">html</p></li>
+            <li class="bootstrap-box"><p class="code-name">bootstrap</p></li>
+            <li class="ruby-box"><p class="code-name">Ruby</p></li>
+          </ul>
+          
+       </div>
+
+      </div>
+
+      
+     
+    </div>
+  </section>
+
   
+    <section class="aboutmyself">
+
+      <div class="aboutme">
```

Comment:
- [ ] Good job on the about me section styling 👍🏽. I think the spacing between the title is too much. I would recommend you reduce it 👍🏽 

Figma Design 👇🏽 

![image](https://user-images.githubusercontent.com/63473584/192101371-9224b715-6899-4bbc-bd76-0c9be7396e04.png)

Your Design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/192101388-770f23c9-636d-4082-bd2c-ab52c1512b67.png)